warehouse/query-core/src/main/java/datawave/query/index/lookup/ShardRangeStream.java

```diff
@@ -1,5 +1,6 @@
 package datawave.query.index.lookup;
 
+import java.util.Collection;
```

Comment:
This import doesn't appear to be used anywhere with this class (Collections is though) so it could be removed